lib/json_schemer/schema/base.rb

```diff
@@ -101,7 +101,7 @@ def validate_instance(instance, &block)
         if keywords
           keywords.each do |keyword, callable|
             if schema.key?(keyword)
-              result = callable.call(data, schema, instance.pointer)
+              result = callable.call(data, schema, instance)
```

Comment:
I think we should use `instance.data_pointer` here instead. Looks like I broke this in: https://github.com/davishmcclurg/json_schemer/commit/d29070324e1549567f88981e25f329929f53f4df

We may want to expose the whole `Instance` object in the future, but for now I'd rather just restore what was there.

Reply:
Thanks for the assistance on the tests. 
I will make the suggested change.